apps/webapp/src/components/magic-link/custom-modal.tsx

```
@@ -0,0 +1,43 @@
+  contentClass = "",
+}) => {
+  if (!open) return null;
+
+  return (
+    <div
+      onClick={() => setOpen(false)}
+      className={`
+        fixed inset-0 flex justify-center items-center transition-colors
+        ${backgroundClass}
+      `}
+    >
```

Biome

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)